### ISSUE TITLE
Readme.md

### ISSUE THREAD

**Issue by jaygarcia** [UXtemple/react-native-es6#1]
Can you put instructions in the Readme.md? 

I installed this module in my project dir and can't tell if it's actually working.

**Comment by dariocravero** [comment]
Yup :) Sorry about that, will do it tomorrow as soon as I have a minute :)
